source/features/show-associated-branch-prs-on-fork.tsx

```diff
@@ -78,26 +76,26 @@ async function init(): Promise<void> {
 			const branchName = branchCompareLink.closest('[branch]')!.getAttribute('branch')!;
 			const prInfo = associatedPullRequests[branchName];
 			if (prInfo) {
-				const StateIcon = prInfo.state === 'MERGED' ? GitMergeIcon : GitPullRequestIcon;
+				const StateIcon = stateIcon[prInfo.state];
```

Comment:
Just asking:

Is there a way to avoid this variable?

Reply:
With a wrapper that's much bigger than this line

Reply:
With a wrapper that's much bigger than this line